.github/workflows/shared-build/action.yml

```diff
@@ -11,15 +11,15 @@ runs:
         echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
         echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
     - name: Setup Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v5
       with:
         python-version: "3.12"
     - name: Setup Node
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v4
       with:
-        node-version: "18.2"
+        node-version: "18.20"
```

Comment:
We should upgrade to Node 20. Please tackle in another PR.

Reply:
Here is the PR: #657